### DISCUSSION TITLE
Using github notes ([!NOTE]) does not seem to work

### DISCUSSION THREAD

**Discussion by flazouh** [remarkjs/remark#1326] [Q&A] [answered]
Hi guys do you have any idea on why this is not working?

> [!NOTE]
> test

Even though it's working on Github ?

Here is my code:

```jsx
import Markdown from 'react-markdown'
import remarkGfm from 'remark-gfm'
const markdowncomponent = () => {
return <Markdown remarkPlugins={[remarkGfm]}>{'> [!NOTE]
> test'}</Markdown>
}
```

https://codesandbox.io/p/sandbox/friendly-wave-g7l78t

Thanks in advance for your help!

**Comment by ChristianMurphy** [comment, marked as answer]
See https://github.com/rehypejs/rehype-github/issues/3

**Reply by flazouh** [reply]
Awesome thanks!
